wp-content/themes/core/assets/css/src/theme/typography/all.pcss

```diff
@@ -15,3 +15,6 @@
 @import "p.pcss";
 @import "quotes.pcss";
 @import "tables.pcss";
+
+/* Themes */
+@import "themes.pcss";
```

Comment:
Nothing to change, for the future, this should get rolled up into whatever theme/skin approach we land on

Reply:
:+1: